parsec/backend/templates/invitation_mail.html

```diff
@@ -458,24 +456,20 @@
                         </tbody>
                       </table>
                       <p>
-
                         {% if is_user_invitation %}
-                        You should then get in touch with <b>{{ greeter }}</b> and follow the next steps on the Parsec
-                        client.
+                        3. Get in touch with <b>{{ greeter }}</b> and follow the next steps on the Parsec client.
                         {% elif is_device_invitation %}
-                        You should then start the invitation process from a device already part of the organization,
+                        3. Start the invitation process from a device already part of the organization,
                         then follow the steps on the Parsec client.
                         {% elif is_shamir_recovery_invitation %}
-                        You should then get in touch with <b>{{ greeter }}</b> and follow the next steps on the Parsec
-                        client.
+                        3. Get in touch with <b>{{ greeter }}</b> and follow the next steps on the Parsec client.
```

Comment:
`parsec/backend/templates/invitation_mail.txt` seems to have conditional but not here

Reply:
Not sure I follow... step 3 appears multiple times because it is inside an if/elif/elif. And is the same for the txt template. What is the suggestion?

Reply:
The suggestion is invalid because I didn't see the `if/elif/else` block :sweat_smile: